### PR TITLE
feat(metric): 🌟 add NIMA

### PR DESCRIPTION
**Pull request by kargaranamir**
First I tried this [code](https://github.com/truskovskiyk/nima.pytorch), but I couldn't find any pre-trained models for it. According to the [code](https://github.com/truskovskiyk/nima.pytorch/blob/bbdbeeb8c22d880205a4fa35cfc2a533d064ee5d/nima/model.py) it uses one of the ResNet models. However, most of the codes on the Internet use the previous version of this repo, and the maintainers of this project [assure](https://github.com/truskovskiyk/nima.pytorch/tree/master#previous-version-of-this-project-is-still-valid-and-works) that the previous version is still valid and working. The [code](https://github.com/truskovskiyk/nima.pytorch/tree/v1
) that I use is based on MobileNetv2, and the pre-trained model URL can be found [here](https://github.com/truskovskiyk/nima.pytorch/blob/v1/settings.ini).

It seems the deployed model on the Heroku app is different from the pre-trained one ([issue8](https://github.com/truskovskiyk/nima.pytorch/issues/28), [issue28](https://github.com/truskovskiyk/nima.pytorch/issues/8))

### CODE REVIEW

**Comment by kargaranamir** [comment]
DENSENET architecture gets a better result according to this [repo](https://github.com/aigagror/ML-Aesthetics-NIMA). Also, we tested NIMA's different architectures on the website aesthetic dataset provided by [labinthewild](http://labinthewild.org/data/index.php) and came to the same conclusion. The current model downloaded from [here](https://github.com/delldu/ImageNima/tree/master/models).

File: backend/aim/metrics/m18/m18_nima.py [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+# ----------------------------------------------------------------------------
+
+
+class Metric(AIMMetricInterface):
+    """
+    Metric: NIMA (Neural IMage Assessment).
+    """
+
+    # Transform method
+    _transform: transforms.transforms.Compose = transforms.Compose(
```
Comment (kargaranamir) [inline backend/aim/metrics/m18/m18_nima.py]: Add details on the square crop.

File: backend/aim/metrics/m18/m18_nima.py [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+# -*- coding: utf-8 -*-
+
+"""
+Metric:
+    NIMA (Neural IMage Assessment)
+
+
+Description:
+    Technical and aesthetic qualities of the input image.
+
```
Comment (kargaranamir) [inline backend/aim/metrics/m18/m18_nima.py:16]: Link about the source.

**Comment by mplaine** [comment]
Minor tasks:

- magic numbers -> constants in m18_nima.py
- assign m18 to a category in metrics.json
- add a score section for m18_1
- run evaluator and update histogram images and score details for m18 in metrics.json
